src/app/components/cards/NotificationsList.jsx

```diff
@@ -212,15 +209,7 @@ class NotificationsList extends React.Component {
                         </Callout>
                     )}
 
-                {loading && (
```

Comment:
Why remove the loading indicator? Would be nice to keep it  to provide feedback on poor connections.

Reply:
This wasn't actually doing anything - vestige of some other loading state. For this component loading state is represented by the `notificationActionPending` prop.

Reply:
Hmm.. it was working before, but it's not on `bridge-api-dev`, so it must have been a regression in another PR